cluster-operator/src/main/java/io/strimzi/operator/cluster/operator/assembly/KafkaAssemblyOperator.java

```diff
@@ -2000,6 +2000,7 @@ String zkConnectionString(int connectToReplicas, Function<Integer, String> zkNod
 
                 ListenerStatus ls = new ListenerStatusBuilder()
                         .withType(listener.getName())
+                        .withName(listener.getName())
```

Comment:
From the community call, it sounded to me like we have two getters and we do not have to call this twice but instead always just call `withName` and it sets both name and type. So I'm a bit confused if this is still needed or not.

Reply:
I thought that on the community call we agreed on that we can have both the `setters` and these setter generate the method `withName` and `withType` which are then being used in the code

Reply:
Ok, maybe I forgot it. But for the record, it is bad implementation because sooner or later, someone will not set one of them and break the API one way or another. If this is our best way to do this, I do not think we should wait for a new CRD version but stop using it much sooner to get rid of this.